src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -88,19 +85,17 @@ public void BuildAppWithSystemNamespace ()
 		}
 
 		[Test]
-		public void DesignTimeBuild ([Values(false, true)] bool isRelease, [Values (false, true)] bool useManagedParser, [Values (false, true)] bool useAapt2)
+		public void DesignTimeBuild ([Values(false, true)] bool isRelease, [Values (false, true)] bool useManagedParser)
```

Comment:
This change implies that `$(AndroidUseAapt2)`=false is no longer supported, but I can't find any mention within Documentation that suggests that aapt2 is now *required*.

@dellis1972?

Reply:
By my understanding `aapt` was never supported in our .NET SDK, some context is here: https://github.com/xamarin/xamarin-android/commit/536a5e780491b52281818f6544f8bacd9cd38576

Reply:
I think we split them into `Xamarin...Aapt.targets` and `Xamarin...Aapt2.targets`, and we never shipped the former in .NET 6+.

Reply:
aapt2 is the only aapt we support since .net 6. You get an error if you try otherwise.